packages/oui-progress-tracker/_mixins.less

```diff
@@ -81,20 +86,37 @@
       }
     }
 
-    &__step_complete {
-      cursor: pointer;
-      pointer-events: initial;
+    &__label {
+      position: relative;
+      top: rem-calc(40);
+      min-width: @label-min-width;
+      max-width: @label-max-width;
+    }
 
+    &__connector {
+      position: relative;
```

Comment:
This property seems useless. You could remove it

Reply:
This class is now removed :)